### ISSUE TITLE
Trucks without an axle defined in the "axles" section are much slower

### ISSUE THREAD

**Issue by whitespace-rebel** [RigsOfRods/rigs-of-rods#3190]
Hi everyone,

recently I discovered that vehicles without an axle defined are much slower than those with an axle defined in the axles section.
Even with fusedrag set to 0.0 and no wings the vehicles struggle to reach realistic maximum speed.

For demonstration you can use this truck:
https://forum.rigsofrods.org/resources/mercedes-lp322-moving-van.348/

Try the original one and remember the maximum speed you have reached.

Then in "mercedes_lp322.truck" add the axles section after the meshwheels secton:
;;;;;;;;;;;;;;;;;
axles
w1(34 36), w2(35 37), d(s)
;;;;;;;;;;;;;;;;;

Accelerate to maximum speed and compare it with your previous maximum speed.

BTW if it is intended behaviour because of compatibility with existing trucks, we can close this bug report.

**Comment by whitespace-rebel** [comment]
I edited my original post because the previous example was not good to show the problem.

**Comment by CuriousMike56** [comment]
Pretty sure this is intentional for backwards compatibility, torque has to be transferred differently on older heavier (10+ ton) mods. 
All vehicles should feature an axles section anyway.

**Comment by whitespace-rebel** [comment]
> All vehicles should feature an axles section anyway.

I agree. Maybe this should be mentioned in the documentation (for new modders).
Right now it says the axles section "allows more accurate distribution of torque among the wheels".
I thought: "As long as I get all the power to the wheels, I dont care much about the distribution among the wheels."
But it turns out some power is lost with the old model.
